docs/concepts/callbacks.md

```diff
@@ -0,0 +1,69 @@
+# Callbacks
+
+Async model endpoints can be configured to send callbacks to a user-defined
+callback URL. Callbacks are sent as HTTP POST requests with a JSON body. The
+following code snippet shows how to create an async model endpoint with a
+callback URL.
+
+To configure an async endpoint to send callbacks, set the `post_inference_hooks`
+field to include
+[`launch.PostInferenceHooks.CALLBACK`](/api/hooks/#launch.hooks.PostInferenceHooks).
+A callback URL also needs to be specified, and it can be configured as a default
+using the `default_callback_url` argument to
+[`launch.LaunchClient.create_model_endpoint`](/api/client/#launch.LaunchClient.create_model_endpoint)
+or as a per-task override using the `callback_url` field of
+[`launch.EndpointRequest`](/api/model_endpoints/#launch.model_predictions.EndpointRequest).
+
+!!! Note
+    Callbacks will not be sent if the endpoint does not have any post-inference
+    hooks specified, even if a `default_callback_url` is provided to the endpoint
+    creation method or if the prediction request has a `callback_url` override.
+
+
+```py title="Creating an Async Model Endpoint with a Callback URL"  hl_lines="16-17 32"
+import os
+import time
+from launch import EndpointRequest, LaunchClient, PostInferenceHooks
+
+client = LaunchClient(api_key=os.getenv("LAUNCH_API_KEY"))
+endpoint = client.create_model_endpoint(
+    endpoint_name="demo-endpoint-callback",
+    model_bundle="test-bundle",
+    cpus=1,
+    min_workers=1,
+    endpoint_type="async",
+    update_if_exists=True,
+    labels={
+        "team": "infra",
```

Comment:
👀

Reply:
Would feel more comfortable with a link to our canonical team list.

Reply:
Oh wait this is a public repo. Hmm...

Do we still require the notion of teams for external?

Reply:
Created https://app.shortcut.com/scaleai/story/727036 to track